### PR TITLE
GitHub Actions: Add Python v3.11 on Django v4.1

### PR DESCRIPTION
**Pull request by cclauss**

### CODE REVIEW

**Comment by sobolevn** [comment]
Can you please drop `safety`?

**Comment by cclauss** [comment]
Taking down the guardrails is not advised.  Better to upgrade the dependencies so users do not rely on an insecure platform.

**Comment by sobolevn** [comment]
I decided to remove `safety` from all of my projects, because I haven't seen a true positive in like 5-6 years of using it. Only noise.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -53,7 +55,7 @@ jobs:
         poetry run doc8 -q docs
         poetry check
         poetry run pip check
-        poetry run safety check --full-report
+        poetry run safety check --full-report || true
```
Comment (sobolevn) [inline .github/workflows/test.yml:58]: Let's remove it :)

Comment (cclauss) [reply .github/workflows/test.yml:58]:
```suggestion
```

**Review by sobolevn — APPROVED** [review]
Thank you!

**Pull request merged**